lib/enhance-assert.js

```diff
@@ -12,23 +13,40 @@ module.exports.PATTERNS = [
 	't.regexTest(regex, contents, [message])'
 ];
 
-function enhanceAssert(assert) {
-	var empower = require('empower');
-	var powerAssertFormatter = require('power-assert-formatter');
-	var powerAssertRenderers = require('power-assert-renderers');
+module.exports.NON_ENHANCED_PATTERNS = [
+	't.pass([message])',
+	't.fail([message])',
```

Comment:
The non-enhanced patterns are a new feature we are adding to power-assert. There will be no source code transform, but we do wrap the function and automatically extract the optional message parameter for you and pass it as `event.originalMessage`. It will be `undefined` if the user does not pass it in.